src/modules/LanItems.rb

```diff
@@ -1386,8 +1320,8 @@ def bridge_slaves(master)
 
     # Creates item's startmode human description
     #
-    # @param item_id [Integer] a key for {#Items}
-    def startmode_overview(item_id)
+    # @param devmap [Hash] Interface sysconfig configuration
```

Comment:
I expect here TODO as overview should not be tight to sysconfig only.

Reply:
ok